modules/installation-initializing.adoc

```diff
@@ -44,8 +44,11 @@ ifeval::["{context}" == "installing-restricted-networks-aws-installer-provisione
 :aws:
 :restricted:
 endif::[]
-ifeval::["{context}" == "installing-azure-customizations"]
-:azure:
+ifeval::["{context}" == "installing-restricted-networks-aws"]
```

Comment:
Two points:
* When there's a new `ifeval`, I expect to see either a new assembly or a change to the `{context}` that is set in an assembly.  I don't see that assembly in this PR.
* Please check me on this, but I don't think the `aws-restricted-network` attribute that is set by this `ifeval` is used in the module.  If it isn't, then I think you can safely remove lines 47-49 and 517-519.

Reply:
I'm not sure about what's being shown in lines 44 to 49 here, but the actual file shows this:
![image](https://user-images.githubusercontent.com/83772007/130242209-4e10b587-60a1-427a-8754-05d8c95466b1.png)

Reply:
- I don't remember what happened here. I didn't do anything with AWS files, so line 47 isn't a new ifeval, so the assembly also already existed.
- I don't see it used, either. Still, since I didn't do anything with AWS, I hesitate to cut those line in case I'm missing something. Do these lines cause damage by being in there? If not, are you OK with leaving them?